talaiot/src/main/kotlin/com/cdsap/talaiot/publisher/JsonPublisher.kt

```diff
@@ -0,0 +1,27 @@
+package com.cdsap.talaiot.publisher
+
+import com.cdsap.talaiot.entities.ExecutionReport
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import org.gradle.api.invocation.Gradle
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileWriter
+
+class JsonPublisher(val gradle: Gradle) : Publisher {
```

Comment:
same as `TimeLinePublisher`, I will add it as default publishers for the configuration

Reply:
Done